release/dist/qnap/files/scripts/Dockerfile.qpkg

```diff
@@ -1,9 +1,24 @@
-FROM ubuntu:20.04
+FROM ubuntu:24.04
 
 RUN apt-get update -y && \
     apt-get install -y --no-install-recommends \
     git-core \
-    ca-certificates
+    ca-certificates \
+    apt-transport-https \
+    gnupg \
+    curl \
+    patch
+
+# Install QNAP QDK
 RUN git clone https://github.com/qnap-dev/QDK.git
+
+# Pin a specific revision of the QDK since we patch it.
+RUN cd /QDK && git reset --hard 18208315614677fc9a6493e90b60f6eb0c90e6e9
```

Comment:
since we have a private patch, could we instead fork the QDK repo and merge the patch over it?

Reply:
The QDK repo is GPL, I didn't want to mess with distributing it.

Reply:
Thinking about it some more, I like forking better. Stay tuned.

Reply:
Okay, this now uses https://github.com/tailscale/QDK